ExpressionTree/ExpressionTree/Operation.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System.Text;
+
+namespace ExpressionTree;
+
+public class Operation
+{
+    public enum Operations
+    {
+        Add,
+        Subtract,
+        Multiply,
+        Divide
+    }
+
+    private string? value;
+
+    public string? Value
+    {
+        get => value;
+        set => this.value = value;
+    }
+
+    private Operations operation;
+
+    public Operation(Operations operation)
+    {
+        this.operation = operation;
+        switch (operation)
+        {
+            case Operations.Add:
+                value = "+";
+                break;
+            case Operations.Subtract:
+                value = "-";
+                break;
+            case Operations.Multiply:
+                value = "*";
+                break;
+            case Operations.Divide:
+                value = "/";
+                break;
+            default:
+                value = null;
+                break;
+        }
```

Comment:
Это можно написать так:
```suggestion
        value = operation switch
        {
            Operations.Add => "+",
            Operations.Subtract => "-",
            Operations.Multiply => "*",
            Operations.Divide => "/",
            _ => null,
        };
```